src/styles/main.scss

```diff
@@ -0,0 +1,32 @@
+@use './blocks/card.scss';
+@use './blocks/stars.scss';
+@use './blocks/reset.scss';
+@use './blocks/navigation.scss';
+@use './blocks/header.scss';
+@use './utils/variables' as *;
+
+@media (max-width: 487px) {
+  .main {
+    grid-template-columns: repeat(1, $column);
+  }
+}
```

Comment:
put all media queries into .main so you don't need to repeat .main inside each of them